cfpb/cfgov-refresh
# Updating testing dependencies

Updating testing dependencies

## Changes

- Modified files to update testing dependencies. 

## Testing

1. Run `frontend.sh` and ensure the build finishes, without error.
2. Run `gulp test:unit` and ensure the build finishes.
3. Run `gulp test:acceptance` and ensure the build finishes.

## Checklist


## Testing checklist

### Browsers


### Accessibility


### Other
